src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('MdAutocomplete', () => {
         }},
         {provide: Dir, useFactory: () => ({value: dir})},
         {provide: ScrollDispatcher, useFactory: () => {
-          return {scrolled: (delay: number, callback: () => any) => {
+          return {scrolled: (_delay: number, callback: () => any) => {
```

Comment:
Why does this need an underscore? I'd prefer not to use `_` for function arguments; is TS not smart enough to know that you've included an argument so that you can have access to a subsequent one?